include/generators/injection.h

```diff
@@ -7,7 +7,7 @@ struct Injection {
   // Particle type
   std::string particle_type;
   // Material id
-  unsigned material_id;
+  std::vector<unsigned> material_ids;
```

Comment:
Any thought of using a vector instead of map?

Reply:
Faster to retrieve

Reply:
I guess so with small data like this